server/src/main/java/com/objectcomputing/checkins/services/team/TeamController.java

```diff
@@ -1,94 +1,126 @@
 package com.objectcomputing.checkins.services.team;
 
-import java.net.URI;
-import java.util.List;
-import java.util.UUID;
-
-import javax.annotation.Nullable;
-import javax.validation.Valid;
-
+import io.micronaut.http.HttpRequest;
 import io.micronaut.http.HttpResponse;
 import io.micronaut.http.MediaType;
-import io.micronaut.http.annotation.Body;
-import io.micronaut.http.annotation.Controller;
-import io.micronaut.http.annotation.Get;
-import io.micronaut.http.annotation.Post;
-import io.micronaut.http.annotation.Put;
-import io.swagger.v3.oas.annotations.tags.Tag;
-import io.micronaut.http.annotation.Produces;
-
+import io.micronaut.http.annotation.Error;
+import io.micronaut.http.annotation.*;
+import io.micronaut.http.hateoas.JsonError;
+import io.micronaut.http.hateoas.Link;
 import io.micronaut.security.annotation.Secured;
 import io.micronaut.security.rules.SecurityRule;
+import io.swagger.v3.oas.annotations.tags.Tag;
 
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
 
-@Controller(TeamController.TEAM_CONTROLLER_PATH)
```

Comment:
Ditching the path variable is fine now, I think...since the only other place it was used is now using the headers to get the location.  I agree on the `IS_ANONYMOUS` change though...that needs to go back to `IS_AUTHENTICATED`.  It means you will have to make use of the users/roles that @chrismckiernan created in your controller tests, but that is as simple as adding a `.basicAuth('MEMBER', 'MEMBER')` to your request creation in most cases.

Reply:
Just the constants instead of hardcoding 'MEMBER' =) but yes.

Reply:
I think in my role service change I made these IS_AUTHENTICATED, but those changes got wiped out when these changes were pushed. He didn't explicitly remove it, just was working of the code before I made my changes. That being said this should be changed as Michael said.

Reply:
@Maroju100 I think getting this back to `IS_AUTHENTICATED` and the `isLead` change in the dto, and you'll be good to go!

Reply:
added IS_AUTHENTICATED and isLead change.